homeassistant/components/technove/coordinator.py

```diff
@@ -0,0 +1,45 @@
+"""DataUpdateCoordinator for TechnoVE."""
+from __future__ import annotations
+
+from technove import Station as TechnoVEStation, TechnoVE, TechnoVEError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_IP_ADDRESS
+from homeassistant.core import CALLBACK_TYPE, HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, LOGGER, SCAN_INTERVAL
+
+
+class TechnoVEDataUpdateCoordinator(DataUpdateCoordinator[TechnoVEStation]):
+    """Class to manage fetching TechnoVE data from single endpoint."""
+
+    config_entry: ConfigEntry
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        *,
+        entry: ConfigEntry,
+    ) -> None:
+        """Initialize global TechnoVE data updater."""
+        super().__init__(
+            hass,
+            LOGGER,
+            name=DOMAIN,
+            update_interval=SCAN_INTERVAL,
+        )
+        self.technove = TechnoVE(
+            entry.data[CONF_IP_ADDRESS], session=async_get_clientsession(hass)
+        )
+        self.unsub: CALLBACK_TYPE | None = None
```

Comment:
Is this doing anything?

Reply:
No, this is also a leftover from copy and paste. Removed.